src/ocean/util/container/pool/model/IAggregatePool.d

```diff
@@ -639,14 +639,31 @@ public abstract class IAggregatePool ( T ) : IPool, IFreeList!(ItemType_!(T))
         Returs:
             true if a and b are identical or false otherwise.
 
-     **************************************************************************/
+    ***************************************************************************/
 
     protected override bool isSame ( Item a, Item b )
     {
         return this.fromItem(a) is this.fromItem(b);
     }
 
-    /**************************************************************************
+    /***************************************************************************
+
+        Checks if item is currently busy.
+
+        Params:
+            item = item to check
+
+        Returs:
+            true if item is currently busy, false otherwise.
+
+    ***************************************************************************/
+
+    protected override bool isBusy ( Item item )
+    {
+        return this.getItemIndex(item) < this.num_busy_;
```

Comment:
looks like this might be correct, but it's implementation detail of `IPool` (one more vote for having it there: https://github.com/gkotian/ocean/blob/aab03da14b6012f85da7fc0cfe888a5878239f8e/src/ocean/util/container/pool/model/IPool.d#L413)